cEpiabm/src/utility/distance_metrics.cpp

```diff
@@ -0,0 +1,28 @@
+#include "distance_metrics.hpp"
+#include <cmath>
+
+
+namespace epiabm
+{
+
+
+    DistanceMetrics::DistanceMetrics()
+    {}
+
+    double DistanceMetrics::Dist(location loc1, location loc2)
+    { 
+        return DistEuclid(loc1, loc2);
+    }
+            
+    double DistanceMetrics::DistEuclid(location loc1, location loc2)
+    {
+        return pow(pow(loc1.first-loc2.first, 2) + pow(loc1.second-loc2.second, 2), 0.5);
+    }
+            
+    double DistanceMetrics::DistCovidsim(location /*loc1*/, location /*loc2*/)
+    {
+        return 2;
```

Comment:
Is this a placeholder?

Reply:
Very much so

Reply:
How easy would it be to add the covidsim implementation, so we can compare to them directly?

Reply:
Okay we can get this in but given this is an unmarked placeholder I will also make an issue and allocate it to you so we don't forget!